color_detect/color_detect.py

```diff
@@ -18,128 +18,128 @@
 upper_blue = np.array([124,255,255])
```

Comment:
Line `11` to line `18` has global variables that are not constant. All global variable should be constant.